docs/source/postprocessing/openPMD.rst

```diff
@@ -0,0 +1,6 @@
+.. _pp-openPMD:
+
+openPMD
+=======
+
+Please see https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Post-processing-and-Visualization for now.
```

Comment:
currently, this page contains a table of content for links to openPMD resources. that's why I linked it instead of a single resource